src/console_reporter.cc

```diff
@@ -46,17 +48,35 @@ bool ConsoleReporter::ReportContext(const Context& context) {
       FLAGS_color_print = false;
   }
 #endif
-  std::string str = FormatString("%-*s %13s %13s %10s\n",
-                             static_cast<int>(name_field_width_), "Benchmark",
-                             "Time", "CPU", "Iterations");
-  GetOutputStream() << str << std::string(str.length() - 1, '-') << "\n";
 
   return true;
 }
 
+void ConsoleReporter::PrintHeader(const Run& run) {
+  std::string str = FormatString("%-*s %13s %13s %10s",
+                             static_cast<int>(name_field_width_), "Benchmark",
```

Comment:
it doesn't look like this was formatted with clang-format. due you mind doing so with Google style?